app/constants/config.js

```diff
@@ -1,15 +1,16 @@
 import { MAINNET } from "./Decrediton";
 
+// Global config constants
 export const DAEMON_ADVANCED = "daemon_start_advanced";
 export const THEME = "theme";
 export const LOCALE = "locale";
 export const NETWORK = "network";
 export const SET_LANGUAGE = "set_language";
-export const UI_ANIMATION = "ui_animations";
+export const UI_ANIMATIONS = "ui_animations";
 export const SHOW_SPV_CHOICE = "show_spvchoice";
 export const SHOW_TUTORIAL = "show_tutorial";
 export const SHOW_PRIVACY = "show_privacy";
-export const ALLOW_EXTERNAL_REQUEST = "allowed_external_requests";
+export const ALLOW_EXTERNAL_REQUESTS = "allowed_external_requests";
```

Comment:
`ALLOWED_EXTERNAL_REQUESTS` is a more descriptive name for this than `ALLOW_EXTERNAL_REQUESTS`